src/main/kotlin/lotto/LottoMain.kt

```diff
@@ -0,0 +1,28 @@
+package lotto
+
+import lotto.domain.LottoGame
+import lotto.domain.LottoTicket
+import lotto.domain.WinnerTicket
+import lotto.util.LottoNumberGenerator
+import lotto.view.InputView
+import lotto.view.OutputView
+
+class LottoMain {
+}
+
+fun main() {
+    val amount = InputView.askPurchaseAmount()
+    val purchaseCount = LottoGame.purchaseTicket(amount)
+    OutputView.printPurchase(purchaseCount)
+    val lottoNumberList = LottoNumberGenerator.generate(purchaseCount)
```

Comment:
```suggestion
    val lottoNumbers = LottoNumberGenerator.generate(purchaseCount)
```
개인적으로는 변수명에 자료형을 명시하는것을 권장하지않아요. 
타입이 변경되었을 경우 유연하게 대처하지 못한다는 단점떄문인데 어떻게 생각하시나요 ㅎㅎ?

Reply:
오 제 생각에는 자바에서는 타입을 변수 앞에 나타나있어서 말씀하신 방식도 괜찮은 것 같은데요!
코틀린에서는 해당 변수가 어떤 자료형인지 변수명으로 알긴 어려워서 변수명에 자료형을 나타내는 것도 괜찮은 것 같은데요!
변수의 타입이 변경되면 변수명도 바꾸면 될 것 같은데요! ㅎㅎ

Reply:
해당 변수가 어떤 자료형인지 알기 어려운 부분에 공감합니다 ㅎㅎ .. 
그래서 요즘에는 변수 선언시 자료형을 선언하곤 합니다 😄 

`val lottoNumbers: List<LottoNumber> = LottoNumberGenerator.generate(purchaseCount)`
알고 계실것 같은데, 변수 선언시 자료형을 선언하는 부분도 공유드릴게요 !